src/lin.py

```diff
@@ -9,8 +53,19 @@ def main():
     argparser.add_argument("genome", type=argparse.FileType('r'))
     argparser.add_argument("reads", type=argparse.FileType('r'))
     args = argparser.parse_args()
-    print(f"Find every reads in {args.reads.name} " +
-          f"in genome {args.genome.name}")
+    genomes = fasta.fasta_parse(args.genome)
+    reads = fastq.fastq_parser(args.reads)
+
+    result = []
+    for r in reads: 
+        for g in genomes:
+            result.append(linear(r, g))
+    
+    for list in result:
+        for line in list:
+            if (line):
+                print(line)
```

Comment:
Why are you collecting all the output in a list and then scanning it to print it? Can't you just print when you have the results? (I can).